index.js

```diff
@@ -62,9 +63,9 @@ const getPrefix = () => {
   return prefix;
 }
 
-function tryNpmPath() {
+function tryWhich(exec) {
   try {
-    return fs.realpathSync(require('which').sync('npm'));
+    return fs.realpathSync(require('which').sync(exec));
```

Comment:
@MorCohenAres I wonder if we should switch to the pure JS `which` library on npm: https://www.npmjs.com/package/which

I've used it in vscode plugins to great effect.

Reply:
@phated  You mean update it to the latest (promise-based) version?
It seems to be already installed, but outdated.

Anyway, I tried keeping my changes here as small as possible since this is a very popular module.
my original goal was to fix `@aws-amplify/cli` which is used by many users as well.
If you think it's necessary to change this, I'll happily add it.

Reply:
Sorry, you are correct. I didn't see the `require` and thought it was looking up the path to system `which`. Ignore me.